generated_code/aderdg.py

```diff
@@ -131,3 +151,57 @@ def addNeighbor(self, generator):
   @abstractmethod
   def addTime(self, generator):
     pass
+
+class ADERDGStandard(ADERDGBase):
```

Comment:
Name is confusing but I've got no better suggestion.
Maybe we should provide a comment for this class that explains its purpose (e.g. linear hyperbolic cons. law of form [...])

Reply:
How about ADERDGLinear or LinearADERDG?

Reply:
Sounds good!